node/tests/AsyncClient.test.ts

```diff
@@ -16,6 +17,8 @@ describe("AsyncClient", () => {
     let server: RedisServer;
     let port: number;
     beforeAll(async () => {
+        Logger.init("error");
+        Logger.setLoggerConfig("error");
```

Comment:
Why default config isn't good?
BTW I think you can omit the second line

Reply:
Its a lot of noise, you dont see anything because we test failures etc, also errors are useless in most of the cases.
Im considering creating something somehow that will output logs just on failed tests, but this is for a separate round and need some planning.

The first line wont affect if the logger was already initiated, its like that by design so user need to explicitly change the config and wont mistakenly have few different init in the code that mistakenly been changed, so config change unintentionally. The order of the tests is not deterministic so i dont know where i should initiate and where i need to config, and if the first log from the core come first, the logger will be on warn. 

But as said, its a patch for making the tests a bit more easy to understand since the logs are unusable with such noise, and its also affect the speed since it log to the console.

Better type of patch, but also a work is to log to file, and extract on failure.
The best one will be some more sophisticated `test mode` that will filter.

Reply:
Oh I see. Thanks, that makes sense